docs/cudf/source/cudf_pandas/usage.md

```diff
@@ -26,6 +26,10 @@ df.groupby("size").total_bill.mean()  # uses the GPU
 df.apply(list, axis=1)                # uses the CPU (fallback)
 ```
 
+```{note}
+If you have already imported or used pandas in your current kernel session, you will need to restart your kernel and run `%load_ext cudf.pandas` as the first command before any pandas imports or usage.
```

Comment:
Nit: Might be good to wrap this across multiple lines to be consistent with the other line lengths in this file